test/lang/test_skip_tokenizer_srt.py

```diff
@@ -0,0 +1,77 @@
+import json
+import unittest
+import os
+import sys
+cur_path = os.path.split(os.path.realpath(__file__))[0]
+sys.path.append(os.path.abspath(os.path.join(cur_path, '../../python')))
+
+import sglang as sgl
+from sglang.test.test_programs import (
+    test_decode_int,
+    test_decode_json_regex,
+    test_expert_answer,
+    test_few_shot_qa,
+    test_mt_bench,
+    test_parallel_decoding,
+    test_regex,
+    test_select,
+    test_stream,
+    test_tool_use,
+)
+from sglang.test.test_utils import DEFAULT_MODEL_NAME_FOR_TEST
+
+
+class TestSRTBackend(unittest.TestCase):
+    backend = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.backend = sgl.Runtime(model_path=DEFAULT_MODEL_NAME_FOR_TEST,
+                                  skip_tokenizer_init=True)
```

Comment:
Can you pass this test? I guess you cannot pass the tests.

Reply:
modified the test and passed by set model_path to my local dir.